src/logger.rs

```diff
@@ -0,0 +1,19 @@
+use std::env;
+use chrono::{DateTime, Utc};
+use std::fs::OpenOptions;
+use std::io::prelude::*;
+
+/// A Logger which writes logs when env-variable RUST_LOG=info.
+#[derive(Debug)]
+pub struct Logger{}
+
+// use flexi_logger::{Logger, opt_format};
+impl Logger {
+    /// Creates a new [`Logger`].
+    ///
+    /// [`Logger`]: struct.Logger.html
+    pub fn log(msg: &str) {
+        let now: DateTime<Utc> = Utc::now();
+        info!("{}: {}\n", now, msg);
+    }
+}
```

Comment:
This is normally implemented by a particular logging implementation. For instance, [`env_logger`](https://docs.rs/env_logger/0.6.2/env_logger/) already prints the time.

I think we do not need a `Logger`. We can use `log` macros directly, once we need them.

Reply:
Ah yeah right, forgot to edit this 🙈